### ISSUE TITLE
Dark Mode doesn't sync with system theme on initial load

### ISSUE THREAD

**Issue by trevorkw7** [acmucsd/membership-portal-ui-v2#38]
Steps to replicate:
1. Set system theme on your device to dark mode
2. Load the page in the browser for the first time or clear localstorage variables and reload the page.

Problem: Website theme doesn't match the preferred theme of the device when the device theme is set to dark mode on initial load.

**Comment by WishingWell13** [comment]
```
Useful console.log statements:

console.log(`Curr theme ${theme?.toString()}`);
  console.log(`Curr theme is undefined ${theme === null}`);
  console.log(`Curr system theme is ${systemTheme?.toString()}`);
  console.log(
    `Toggled should be ${theme === 'system' ? systemTheme === 'dark' : theme === 'dark'}`
  );
```

**Comment by SheepTester** [comment]
I think this was fixed in #44
